README.md

```diff
@@ -68,18 +68,22 @@ jobs:
           directories: "components/my_component;components/another_component"
           namespace: "espressif"
           api_token: ${{ secrets.IDF_COMPONENT_API_TOKEN }}
+          send_source_directory: "False"
```

Comment:
Could you please make a separate example to show the case of using a separate repo with source

Reply:
I deleted this parameter, as we discussed in the Mattermost